gateway/src/main/java/io/camunda/zeebe/gateway/impl/probes/health/ResponsiveHealthIndicator.java

```diff
@@ -48,17 +48,16 @@ HealthZeebeClientProperties getHealthZeebeClientProperties() {
   @Override
   public Health health() {
     Builder resultBuilder;
-    try (final var zeebeClient = supplyZeebeClient()) {
-
-      if (zeebeClient == null) {
-        resultBuilder = Health.unknown();
-      } else {
-        try {
-          zeebeClient.newTopologyRequest().send().get();
-          resultBuilder = Health.up();
-        } catch (final Throwable t) {
-          resultBuilder = Health.down().withException(t);
-        }
+
+    initZeebeClient();
```

Comment:
@remcowesterhoud may I ask why we have the client usage in the gateway? Do you know that? Feels unnecessary dependency and request we sent?